pkg/sensors/tracing/args.go

```diff
@@ -34,6 +34,7 @@ const (
 	argReturnCopyBit    = 1 << 4
 	argMaxDataBit       = 1 << 5
 	argUserspaceDataBit = 1 << 6
+	argRawSyscallsBit   = 1 << 7
```

Comment:
The idea was to prepare for being able to filter on arguments in raw_syscalls, and therefore arrays of arguments which could potentially be pointers. I thought it was a bit much to include that in this PR, but I was already thinking ahead for a following PR. Can do as you suggest if you prefer though.

Reply:
> Can do as you suggest if you prefer though.

Thanks! I would prefer to keep the code as simple as possible, even if we end up extending this for filtering.

I've also thought about filtering `raw_syscalls`.

One way would be to do as you suggest, and use the arguments but this seems tricky because we need to pass types for everything.

I think a better approach would be to do two-level filtering where we in  `raw_syscall` event we  to filter the type of the syscall and then we enable hooks on syscalls/ for filtering on the arguments. This seems simpler to me and it re-uses the arg filtering we already have. We would need to have a way to pass information from the `raw_syscall` hook to the specific syscall hook, but we already have something similar Jiri's notify enforcer action.

In any case, that's for another PR.